### PR TITLE
3.2

### PR DESCRIPTION
**Pull request by ClinToch**
**Visual Scriting Editor**

hello everyone, my feedback goes to the Visual Scriting Editor in Godot Game Engine.

As an Artist, I love the idea of visual scripting as it enables me to script my game in a visual way without having all the technical knowledge of programming and its challenges.

Godot 3.1 was good in terms of the arrangement of nodes in a grouped manner(this made it easy to find the nodes that you where looking for) but now 3.2 has gone ahead to make them ungrouped and separate on their own making it less of an option for beginners and Artist alike who may never want to learn how to program.

to sum this up I think we need to give Artist the voice to make the wise decisions on making Visual Scriting in Godot better (especially for the Artist).

thanks

### CODE REVIEW

**Comment by clayjohn** [comment]
It seems you have mistakenly opened a pull request. Pull requests are how you contribute engine changes back to the main Godot branch. Pull requests are absolutely not a place to provide general feedback.

If you would like to provide general feedback, consider starting a discussion on one of the various [Community Channels](http://docs.godotengine.org/en/latest/community/channels.html)

If you have an idea for something that could improve the engine, consider submitting a proposol to [Godot-Proposals](https://github.com/godotengine/godot-proposals)
